lib/cmd.js

```diff
@@ -62,17 +64,17 @@ class Cmd {
             } else {
               //slightly different assignment of name since it comes from child prompt
               if (options.simple) {
-                embark.generateTemplate('simple', './', inputvalue);
+                this.embark.generateTemplate('simple', './', inputvalue);
```

Comment:
Does `this` work here? Since we are in another function, I'd expect `embark` to be undefined. Might need to use `self`.

Reply:
In fact, all `this`s would need self in this function, no?